script/jenkins/bench.sh

```diff
@@ -1,9 +1,4 @@
 #!/usr/bin/env bash
 set -euxo pipefail
 
-source ./_beats/dev-tools/common.bash
-
-jenkins_setup
-
-go get -u golang.org/x/tools/cmd/benchcmp
```

Comment:
Where is this vendored/installed now?

Reply:
the command runs in a Docker container with all the tools installed